demo/oc_generate_apps2.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#Test oc tool
+#Assumes User is logged in to cluster
+#Assumes Ansible is installed on machine
+
+# Will create nginx deployment
+cd ~/projects/container-pipelines/basic-nginx
```

Comment:
Let's make this path an argument passed to the script.

Reply:
Done.